cfpb/consumerfinance.gov
# Use react-select instead of chosen for agreements db

The credit card agreement database use(d) `chosen` and v3.5.1 of `jquery`, both included as static files instead of true frontend dependencies.

This PR makes an actual frontend app for the agreement db and switches off the jquery-powered `chosen` to `react-select`.

## Testing
 - visit http://localhost:8000/credit-cards/agreements/
 - The select should function identically to https://www.consumerfinance.gov/credit-cards/agreements/
 - Go to any agreements page, eg, http://localhost:8000/credit-cards/agreements/issuer/1st-century-bank-national-association/
 - The select in the sidebar should also function identically to prod
